tests/devices/qubit_mixed/test_qubit_mixed_sampling.py

```diff
@@ -175,27 +175,6 @@ def test_invalid_state(self):
         ):
             sample_state(invalid_state, 10)
 
-    def test_measure_with_samples_not_implemented_error(self):
```

Comment:
Are we deleting this because the error message is changed or because now measures with samples are supported?

Reply:
yes. This is a missing point from last PR